R/fct_db.R

```diff
@@ -0,0 +1,28 @@
+#' db 
+#'
+#' @description A fct function
+#'
+#' @return The return value, if any, from executing the function.
+#' @importFrom DBI dbConnect
+#' @importFrom RPostgres Postgres
+#'
+#' @noRd
+connect <- function(
+  session = shiny::getDefaultReactiveDomain()
+){
+  
+  session$userData$con <- dbConnect(
```

Comment:
Je ne peux pas tester la fonction en l'état sans être dans une session Shiny. Si je fais `get_con()` le résultat est NULL.  
Merci de mettre à jour la vignette à préfabriquer "dev/aa-data-exploration-and-preparation.Rmd" pour pouvoir utiliser `connect()` et `get_con()`, puis fermer la connexion à la base.  

Je suppose qu'il faut quelque chose comme ça pour exemple lorsque la BDD existe :
```r
session <- new.env()
connect(session)
get_con(session)
# <PqConnection> diades@localhost:5432
DBI::dbDisconnect(get_con(session))
```

Reply:
Solved in https://github.com/inrae/diades.atlas/pull/30/commits/9e2f43c733dacec2832ac7bfd2241520fefc4a1c